geometry_msgs/Vector3 position
geometry_msgs/Quaternion orientation
---
bool success
string message